core/src/main/java/edu/wpi/grip/core/sources/CameraSource.java

```diff
@@ -154,8 +154,21 @@ public void start() throws IOException, IllegalStateException {
 
                     frameMat.copyTo(frameOutputSocket.getValue().get());
                     frameOutputSocket.setValue(frameOutputSocket.getValue().get());
-                    long thisMoment = System.currentTimeMillis();
-                    frameRateOutputSocket.setValue(1000 / (thisMoment - lastFrame));
+
+                    final long thisMoment = System.nanoTime();
+                    final long elapsedTime = thisMoment - lastFrame;
+                    if (elapsedTime == 0) {
+                        /* This can happen when using system.currentTimeMillis():
+                         * https://github.com/WPIRoboticsProjects/GRIP/issues/199
+                         * As such we use system.nanoTime().
+                         * However, we add an assertion to ensure this bug doesn't reappear.
+                         */
+                        assert elapsedTime != 0 : "The time between grabbing two frames was less than 1 nanosecond";
```

Comment:
What do you want me to do instead then? Leave it as a divide by zero error but using nano time instead? What do you think should be here?